pages/opam-packaging.html

```diff
@@ -103,13 +173,13 @@ <h2>Submitting your new package</h2>
 <p>Submission happens by <a href="https://help.github.com/articles/creating-a-pull-request/">creating a pull request</a>
  on the <a href="https://github.com/coq/opam-coq-archive">opam-coq-archive repository</a>.</p>
 
-<p>First push your changes to you personal fork</p>
+<p>First push your changes to you personal fork:</p>
 
 <pre><code>git push origin master
```

Comment:
Also, it would be good for this tutorial to include a step of creating a topic branch.